packages/esm-patient-medications-app/src/config-schema.ts

```diff
@@ -30,6 +30,12 @@ export const configSchema = {
     _default: 99,
     _description: 'The maximum number of days for medication dispensing.',
   },
+  debounceDelayInMs: {
+    _type: 'integer',
+    _description:
+      'Number of milliseconds to delay the search operation in the drug search input by after the user starts typing. The useDebounce hook delays the search by 500ms by default',
+    _default: 500,
```

Comment:
Sorry, @jwnasambu, I should've said the default was 300ms. Could you change it to say that in both the `default` and the `description` keys above?